tests/message-extensions/src/index.ts

```diff
@@ -0,0 +1,129 @@
+import { cardAttachment } from '@microsoft/teams.api';
+import { App } from '@microsoft/teams.apps';
+import { Card } from '@microsoft/teams.cards';
+import { ConsoleLogger } from '@microsoft/teams.common/logging';
+import { DevtoolsPlugin } from '@microsoft/teams.dev';
+import {
+  createCard,
+  createConversationMembersCard,
+  createDummyCards,
+  createLinkUnfurlCard,
+  createMessageDetailsCard,
+} from './card';
+
+const app = new App({
+  logger: new ConsoleLogger('@tests/message-extensions', { level: 'debug' }),
+  plugins: [new DevtoolsPlugin()],
+});
+
+app.on('message', async ({ send, activity }) => {
```

Comment:
Consider responding with what this app does

Reply:
added `install.add` activity that will send a greeting message on install